android/src/main/java/bbct/android/common/activity/About.java

```diff
@@ -57,4 +60,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return view;
     }
 
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setHasOptionsMenu(true);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int itemId = item.getItemId();
+        if (itemId == android.R.id.home) {
+            getActivity().onBackPressed();
+        }
+        return super.onOptionsItemSelected(item);
+    }
```

Comment:
Can this be moved into `MainActivity` so that it is only implemented once instead of requiring it in every fragment?